command/agent/consul/client.go

```diff
@@ -0,0 +1,678 @@
+package consul
```

Comment:
Can you add metrics to this package that output the number of services/checks/script checks being registered and one on each sync failure.